atuin-client/src/history.rs

```diff
@@ -62,10 +86,209 @@ impl History {
             session,
             hostname,
             deleted_at,
+            _seal: HistorySeal,
         }
     }
 
+    /// Builder for a history entry that is imported from shell history.
```

Comment:
here and below are new builders, with some documentation (and doctests) considering their scenarios and usage